docs/source/backend.rst

```diff
@@ -3,44 +3,88 @@
 torchaudio.backend
 ==================
 
+Overview
+~~~~~~~~
+
 :mod:`torchaudio.backend` module provides implementations for audio file I/O, using different backend libraries.
 To switch backend, use :py:func:`torchaudio.set_audio_backend`. To check the current backend use :py:func:`torchaudio.get_audio_backend`.
 
-.. warning::
-   Although ``sox`` backend is default for backward compatibility reason, it has a number of issues, therefore it is highly recommended to use ``sox_io`` backend instead. Note, however, that due to the interface refinement, functions defined in ``sox`` backend and those defined in ``sox_io`` backend do not have the same signatures.
+There are currently four implementations available.
 
-.. note::
-   Instead of calling functions in :mod:`torchaudio.backend` directly, please use ``torchaudio.info``, ``torhcaudio.load``, ``torchaudio.load_wav`` and ``torchaudio.save`` with proper backend set with :func:`torchaudio.get_audio_backend`.
+    * ``"sox"`` (deprecated)
+    * ``"sox_io"``
+    * ``"soundfile"``, legacy interface (deprecated)
+    * ``"soundfile"``, new interface
+
+``"sox"`` backend is the original backend which is built on ``libsox``. This backend requires C++ extension module and is not available on Windows system. This backend is currently the default backend when available, but is being deprecated. Please use ``"sox_io"`` backend.
 
-There are currently three implementations available.
+``"sox_io"`` backend is the new backend which is built on ``libsox``, and bound with ``TorchScript``. This backend requires C++ extension module and is not available on Windows system. Function calls to this backend are TorchScript-able. This backend will become the default backend in ``"0.8.0"`` release.
 
-    * :ref:`sox<sox_backend>`
-    * :ref:`sox_io<sox_io_backend>`
-    * :ref:`soundfile<soundfile_backend>`
+``"soundfile"`` backend is built on ``PySoundFile``. You need to install ``PySoundFile`` separately. The current interface of this backend (legacy interface) will be changed in ``0.8.0`` (new interface) to match the ``"sox_io"`` backend.
```

Comment:
"You need to install ``PySoundFile`` separately." - how do I get it?

Reply:
"The current interface of this backend (legacy interface) will be changed in ``0.8.0`` (new interface) to match the ``"sox_io"`` backend." - More detail in the corresponding issue? Which one is it?